hourly-tips/src/test/java/org/apache/flink/training/exercises/hourlytips/HourlyTipsTest.java

```diff
@@ -72,27 +84,39 @@ public void testMaxAcrossDrivers() throws Exception {
         TaxiFare tenFor1In2 = testFare(1, t(90), 10.0F);
         TaxiFare twentyFor2In2 = testFare(2, t(90), 20.0F);
 
-        TestFareSource source =
-                new TestFareSource(oneFor1In1, fiveFor1In1, tenFor1In2, twentyFor2In2);
+        ParallelTestSource<TaxiFare> source =
+                new ParallelTestSource<>(oneFor1In1, fiveFor1In1, tenFor1In2, twentyFor2In2);
 
         Tuple3<Long, Long, Float> hour1 = Tuple3.of(t(60).toEpochMilli(), 1L, 6.0F);
         Tuple3<Long, Long, Float> hour2 = Tuple3.of(t(120).toEpochMilli(), 2L, 20.0F);
 
-        List<Tuple3<Long, Long, Float>> expected = Arrays.asList(hour1, hour2);
-
-        assertEquals(expected, results(source));
+        assertThat(results(source)).containsExactlyInAnyOrder(hour1, hour2);
     }
 
     private Instant t(int minutes) {
-        return Instant.parse("2020-01-01T12:00:00.00Z").plusSeconds(60 * minutes);
+        return Instant.parse("2020-01-01T12:00:00.00Z").plusSeconds(60L * minutes);
```

Comment:
Do you think, it's worth putting `Instant.parse("2020-01-01T12:00:00.00Z")` into a static constant not to run it again and again?